packages/next/telemetry/trace/report/index.ts

```diff
@@ -0,0 +1,27 @@
+import { TARGET, SpanId } from '../shared'
+import reportToConsole from './to-console'
+import reportToZipkin from './to-zipkin'
+import reportToTelemetry from './to-telemetry'
+
+export const noop = (
+  _spanName: string,
+  _duration: number,
+  _timestamp: number,
+  _id: SpanId,
+  _parentId?: SpanId,
+  _attrs?: Object
+) => {}
+
+const target =
```

Comment:
It'll actually default to telemetry target if `CONSOLE` or `ZIPKIN` are not specified.  That seemed like a sane default behavior, since we don't expect the env var to be set in the wild (and we want telemetry reporting).  However, that's not clear because of the `noop` definition, which was left as an artifact of development.

Reply:
Also added console output when `TRACE_TARGET` is set but invalid.